### PR TITLE
Add Russian translate

### PR DESCRIPTION
**Pull request by OFGING**

### CODE REVIEW

File: app/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -0,0 +1,163 @@
+<resources>
+    <string name="app_name">GPTMobile</string>
+    <string name="gpt_mobile_introduction_logo">Логотип приветствия GPT Mobile</string>
+    <string name="welcome_title">Лучший AI-помощник,\nкоторый вы можете получить\nна свой смартфон.</string>
+    <string name="welcome_description">Используйте модель, которая вам больше всего подходит.\nЗадавайте любые вопросы!</string>
+    <string name="get_started">Начать</string>
+    <string name="platform_select_description">Выберите платформу, которую хотите использовать. Вы сможете изменить это позже в настройках. В будущем будет поддерживаться больше платформ.</string>
+    <string name="sample_item_title">Элемент 1</string>
+    <string name="sample_item_description">Описание элемента</string>
+    <string name="openai" translatable="false">OpenAI</string>
```
Comment (Taewan-P) [inline app/src/main/res/values-ru/strings.xml]: Please remove all string value that is marked as translatable=false.
For translated string files, there is no need to put them again

**Review by Taewan-P — CHANGES_REQUESTED** [review]
Hi, thanks for the pull request!
Please fix the points that I requested and it will be all set!

**Comment by OFGING** [comment]
I deleted the lines containing translatable=false.

**Review by Taewan-P — APPROVED** [review]
LGTM!

**Pull request merged**